go.mod

```
@@ -1,7 +1,8 @@
 module github.com/fastly/terraform-provider-fastly
 
-go 1.22.0
-toolchain go1.23.7
+go 1.23.0
+
+toolchain go1.23.6
```

```suggestion
toolchain go1.23.7
```

1.23.7 includes security fixes that are relevant for this application.